lib/workorder-form/workorder-form-controller.js

```diff
@@ -66,7 +65,7 @@ function WorkorderFormController($scope, mediator, workorderMediatorService, $st
 
       createUpdatePromise.then(function() {
         //Finished with the update/create, go back to the list.
-        mediator.publish("wfm:ui:workorder:list");
+        mediator.publish('wfm:ui:workorder:selected', workorderToCreate);
```

Comment:
[INFO] We should avoid using mediator for communicating inside module. 
This particular use case is ok, but with the new aproach we should just have change state code here.